custom/icds_reports/tasks.py

```diff
@@ -323,20 +333,32 @@ def _aggregate_gm_forms(state_id, day):
 
 
 @track_time
-def _aggregate_child_health_pnc_forms(state_id, day):
+def _aggregate_pnc_forms(state_id, day):
     AggregateChildHealthPostnatalCareForms.aggregate(state_id, day)
+    AggregateCcsRecordPostnatalCareForms.aggregate(state_id, day)
```

Comment:
If this is split up between `_aggregate_child_health_pnc_forms` and `_aggregate_ccs_record_pnc_forms`, then they will happen in parallel